### DISCUSSION TITLE
Forward nodes to specific search nodes

### DISCUSSION THREAD

**Discussion by Jaap79** [Security-Onion-Solutions/securityonion#8654] [Unsupported Versions] [answered]
Hi,

We are currently running a distributed deployment for SO 2.3.150. I was wondering, is it in any way possible to have forward nodes index their data to specific search nodes?

(Like, location A goes to Search Node #01, location B goes to Search Node #02, et cetera and so forth).

We would like to seperate data from various forward nodes to different search nodes (or, eventually, perhaps groups of search nodes!).

**Comment by dougburks** [comment, marked as answer]
> is it in any way possible to have forward nodes index their data to specific search nodes?

Not really. The closest option would be a heavy node, which we don't recommend for most users due to performance issues:
https://docs.securityonion.net/en/2.3/architecture.html#distributed
